src/libs/actions/IOU.ts

```diff
@@ -480,6 +480,15 @@ type CreateTrackExpenseParams = {
     transactionParams: TrackExpenseTransactionParams;
 };
 
+type GetTrackExpenseInformationParams = {
+    parentChatReport: OnyxEntry<OnyxTypes.Report>;
+    moneyRequestReportID?: string;
+    existingTransactionID?: string;
+    participantParams: RequestMoneyParticipantParams;
+    policyParams: RequestMoneyPolicyParams;
+    transactionParams: RequestMoneyTransactionParams;
```

Comment:
Sorry for back and forth. After checking again I think we should have an explicit type definition for each field i.e. `GetTrackExpenseInformationParticipantParams`, etc.

Sharing other types that are not exactly the same could be confusing and may be hard to maintain if a change is needed.